tests/scrape/test_scrape_stats_league.py

```
@@ -40,6 +40,7 @@ def test_unknown_league_id_raises() -> None:
 
 def _fake_endpoint_span(season: str = "", league_id: str = "") -> None: ...
 def _fake_endpoint_year(season_year: str = "", league_id: str = "") -> None: ...
+def _fake_endpoint_year_nullable(season_year_nullable: str = "", league_id: str = "") -> None: ...
```

Flake8 (7.3.0)

[error] multiple statements on one line (def)

(E704)